[2024-05-21T18:41:35.139+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T18:41:35.153+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-21T18:41:32.962162+00:00 [queued]>
[2024-05-21T18:41:35.158+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-21T18:41:32.962162+00:00 [queued]>
[2024-05-21T18:41:35.158+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-21T18:41:35.167+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): create_db> on 2024-05-21 18:41:32.962162+00:00
[2024-05-21T18:41:35.174+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=63) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T18:41:35.175+0000] {standard_task_runner.py:63} INFO - Started process 67 to run task
[2024-05-21T18:41:35.175+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'games', 'create_db', 'manual__2024-05-21T18:41:32.962162+00:00', '--job-id', '550', '--raw', '--subdir', 'DAGS_FOLDER/get_main_games.py', '--cfg-path', '/tmp/tmpcttkua4x']
[2024-05-21T18:41:35.177+0000] {standard_task_runner.py:91} INFO - Job 550: Subtask create_db
[2024-05-21T18:41:35.206+0000] {task_command.py:426} INFO - Running <TaskInstance: games.create_db manual__2024-05-21T18:41:32.962162+00:00 [running]> on host 8bf53c205e4b
[2024-05-21T18:41:35.261+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='games' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-21T18:41:32.962162+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-21T18:41:32.962162+00:00'
[2024-05-21T18:41:35.262+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T18:41:35.281+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2024-05-21T18:41:36.487+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T18:41:36.487+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T18:41:36.494+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=games, task_id=create_db, execution_date=20240521T184132, start_date=20240521T184135, end_date=20240521T184136
[2024-05-21T18:41:36.510+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T18:41:36.528+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T18:41:36.528+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
